.github/ISSUE_TEMPLATE/01_general_issue_template.yml

```diff
@@ -0,0 +1,27 @@
+name: General
+description: File an issue that is not covered by other issue templates
+title: "[General]:"
+labels: [general]
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to fill out this form!
+
+- type: input
+  id: contact
```

Comment:
This would normally happen directly in the form of a discussion in the project issue tracker - no separate contact information required.
(also, I wouldn't put any in, if I had to fill in this form - that's what the github communication channels are for)

Reply:
We would prefer to get real names and affiliations to have alternate ways of contacting. That's why we want to keep the contact form.